pkg/cloudprovider/vm.go

```diff
@@ -124,5 +124,8 @@ func (i *vmIdentityInfo) AppendUserIdentity(id string) bool {
 }
 
 func (i *vmIdentityInfo) GetUserIdentityList() []string {
+	if i.info.IdentityIds == nil {
```

Comment:
Can we also check if i.Info is nil as well before accessing IdentityIds.

Reply:
We already check that in here - https://github.com/Azure/aad-pod-identity/blob/master/pkg/cloudprovider/cloudprovider.go#L163-L165